pnpm-lock.yaml

```diff
@@ -6530,47 +4020,6 @@ packages:
       handlebars-wax: 6.1.0
     dev: false
 
-  /jsonresume-theme-caffeine@1.2.3:
-    resolution: {integrity: sha512-SCcKReZjSauVWIs55B2NwU4X44iWYBnJuvC93CcjfXDiHBLkIVtZ1OgufgHxm7UD2bSXu4juD/lR+vb9s8EOiA==}
-    engines: {node: '>=4.0.0'}
-    dependencies:
-      address-format: 0.0.3
-      handlebars: 4.7.7
-      handlebars-wax: 4.0.4
-      moment: 2.29.4
-      swag: 0.7.0
-    dev: false
-
-  /jsonresume-theme-class@0.1.2:
-    resolution: {integrity: sha512-xzeDTFJek/F4GE+hE+Y7/vuEzGP7JTSORykuhawdDKbjgstXahTeASDAYt6MoTd4igrTPd8G5HAn+/zhVfV7SQ==}
-    dependencies:
-      handlebars: 2.0.0
-    dev: false
```

Comment:
Here is a nice example. It should be possible to just use `handlebars@^4` but nobody has cared to update the [theme](https://www.npmjs.com/package/jsonresume-theme-class) in 9 years. The alleged github repo holding the code has also gone.